docs/index.rst

```diff
@@ -922,6 +922,28 @@ to properly set types before the model can be used.
 
    Resolving this is beyond the call for *pydantic*: either remove the future import or declare the types globally.
 
+Usage of ``Union`` in Annotations and Type Order
+------------------------------------------------
+
+The ``Union`` type allows a model attribute to accept different types, e.g.:
+
+.. literalinclude:: examples/union_type_incorrect.py
```

Comment:
```suggestion
**(This script is complete, it should run but may be is wrong, see below)**

.. literalinclude:: examples/union_type_incorrect.py
```

I'm bad at reading docs, I search, find the first example that looks write and go with it. Hopefully this message will help other people as lazy as me.

Reply:
makes sense